qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/identifier/OfferId.groovy

```diff
@@ -0,0 +1,20 @@
+package life.qbic.portal.portlet.offers.identifier
+
+/**
+ * <class short description - 1 Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since <versiontag>
+ */
```

Comment:
description missing

Reply:
```suggestion
/**
 * Describes the offer identifier
 *
 * Contains the random part, the project conserved part and the version from which the offer identifier is build
 *
 * @since 0.1.0
 */
```